.github/workflows/api.yaml

```diff
@@ -16,7 +16,7 @@ jobs:
   build:
     uses: VeryGoodOpenSource/very_good_workflows/.github/workflows/dart_package.yml@v1
     with:
-      dart_sdk: 3.4.3
+      dart_sdk: 3.5.0
```

Comment:
Should this be 3.5.2 since that is the Dart version for Flutter 3.24.2?

Reply:
yes, good catch